backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/ddl/functions/scramble_metasfresh.sql

```diff
@@ -1,7 +1,7 @@
 
 --## Function to scramble all metasfresh-tables
 
-CREATE OR REPLACE FUNCTION public.scramble_metasfresh(
+CREATE OR REPLACE FUNCTION ops.scramble_metasfresh(
```

Comment:
Please add a folder `src/main/sql/postgresql/ddl/public` for and move the existing stuff to it.
Then add a folder `src/main/sql/postgresql/ddl/ops/functions` and move the scramble stuff for that.
(I.e. one folder per schema)

Reply:
done